src/UbiProtocol.h

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright (c) 2013-2018 Ubidots.
+Permission is hereby granted, free of charge, to any person obtaining
+a copy of this software and associated documentation files (the
+"Software"), to deal in the Software without restriction, including
+without limitation the rights to use, copy, modify, merge, publish,
+distribute, sublicense, and/or sell copies of the Software, and to
+permit persons to whom the Software is furnished to do so, subject to
+the following conditions:
+The above copyright notice and this permission notice shall be
+included in all copies or substantial portions of the Software.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+Developed and maintained by Jose Garcia and Cristian Arrieta for IoT Services
+Inc
+@jotathebest at github: https://github.com/jotathebest
+@crisap94 at github: https://github.com/crisap94
+*/
+
+#ifndef _UbiProtocol_H_
+#define _UbiProtocol_H_
+
+#include <WiFiNINA.h>
+#include <WiFiUdp.h>
```

Comment:
why do you need udp here?

Reply:
There is no UDP client inside the WiFiNINA library, WifiNINA package comes with the two libraries in separated files